projects/demo/src/modules/components/preview/examples/1/index.less

```diff
@@ -0,0 +1,19 @@
+@import 'taiga-ui-local';
+
+.content {
+    .text-body-xl();
+    background-color: rgb(242, 242, 242);
```

Comment:
```suggestion
    background-color: var(--tui-base-01);
```

to improve dark mode:

<img width="1284" alt="example1" src="https://user-images.githubusercontent.com/35179038/140048389-660c1a79-069e-4488-ac56-491c63d7dadf.png">

Reply:
thanks!